website/templates/website/event.html

```diff
@@ -34,7 +35,8 @@ <h3 class="event-detail">Workshop Schedule</h3>
             <tbody>
               <tr class="table-active">
                 <td>{{ workshop.name }}</td>
-                <td>{{ workshop.time }}</td>
+                <td>{{ workshop.start_time|date:"D d F" }}</td>
+                <td>{{ workshop.start_time|time:"g:i" }} - {{workshop.end_time|time:"g:ia"}}</td>
```

Comment:
Both the start time and end time should use "g:ia". Otherwise, this could happen and cause confusion:
![image](https://user-images.githubusercontent.com/7235102/45250397-a6748600-b375-11e8-8caa-ed2e0d99652d.png)